object_hasher.go

```diff
@@ -54,7 +62,7 @@ func (hasher *objectHasher) HashProto(pb proto.Message) ([]byte, error) {
 	// Make sure the proto itself is actually valid (ie. can be marshalled).
 	// If this fails, it probably means there are unset required fields or invalid
 	// values.
-	if _, err := proto.Marshal(pb); err != nil {
+	if _, err = proto.Marshal(pb); err != nil {
```

Comment:
There is no reason to remove the shadowing of the `err` value here.

Reply:
I agree that keeping the shadowing here would be harmless, but I prefer to avoid shadowing where it's not adding value.

Reply:
My interest here is “if it ain’t broke, don’t fix it”. If you were working on this line anyways, then, meh.

But since the code is behaviorally equivalent…